astroquery/ipac/nexsci/nasa_exoplanet_archive/core.py

```diff
@@ -370,14 +372,14 @@ def query_object_async(self, object_name, *, table="ps", get_query_payload=False
         return self.query_criteria_async(table, get_query_payload=get_query_payload, cache=cache, **criteria)
 
     @class_or_instance
-    def query_aliases(self, object_name, *, cache=None):
+    def query_aliases(self, object_name):
```

Comment:
why remove `cache`? If remove, clear out the docstring, too

Reply:
Wasn't thinking, will add it back.